### ISSUE TITLE
How I can generate idcards.json for different cards

### ISSUE THREAD

**Issue by tamal-appsbee** [tzununbekov/go-idmatch#46]

**Comment by tzununbekov** [comment]
@tamal-appsbee new card templates must be added manually and filled with values based on card individual parameters (aspect ratio, font size, card structure, etc)

**Comment by tamal-appsbee** [comment]
@tzununbekov 

Is there any tools or lib to generate it ?

Thanks

**Comment by tzununbekov** [comment]
@tamal-appsbee no, there are no tools or lib to generate them, templates should be created manually
